pwiz_tools/Skyline/TestTutorial/AbsoluteQuantTutorialTest.cs

```diff
@@ -51,7 +50,7 @@ public void TestAbsoluteQuantificationTutorial()
             // Set true to look at tutorial screenshots.
             // IsPauseForScreenShots = true;
 
-            ForceMzml = (Program.PauseSeconds == 0);   // Mzml is ~8x faster for this test.
+            ForceMzml = true;   // Mzml is ~8x faster for this test.

```

Comment:
This will force mzml files to be used during manual execution and show up in the screenshots. What was the point of this change?